ArduPlane/mode_autoland.cpp

```diff
@@ -129,9 +129,10 @@ bool ModeAutoLand::_enter()
     // Try and minimize loiter radius by using the smaller of the waypoint loiter radius or 1/3 of the final WP distance
     const float loiter_radius = MIN(final_wp_dist * 0.333, fabsf(plane.aparm.loiter_radius));
 
-    // corrected_loiter_radius is the radius the vehicle will actually fly, this gets larger as altitude increases.
-    // Strictly this gets the loiter radius at the current altitude, really we want the loiter radius at final_wp_alt.
-    const float corrected_loiter_radius = plane.nav_controller->loiter_radius(loiter_radius);
+    // corrected_loiter_radius is the radius the vehicle will actually fly, this gets larger as altitude increases
+    const float corrected_loiter_radius =
+        plane.nav_controller->calc_corrected_loiter_radius(loiter_radius, NAN,
```

Comment:
Not a fan of passing nans.

Reply:
But your not actually using all those combinations. I think its fine to even not pass altitude to start with, that is a existing issue that could be fixed in a separate PR.